### PR TITLE
Profile page cleaned

### PR DESCRIPTION
**Pull request by bojanmitic**
Moved all profile files in components, all profile components are in page templates that follow consistency.

### CODE REVIEW

File: src/store/actions/profileActions.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 import { FETCH_USER_PROFILE } from './types';
 
-const fetchUserProfile = userId => dispatch => axios.get(`/api/v1/profiles/${userId}`).then((res) => {
+const fetchUserProfile = () => dispatch => axios.get('https://integrify.network/api/v1/profiles/5b7c5ade5f49453eecccf351').then((res) => {
```
Comment (trungh13) [inline src/store/actions/profileActions.js]: We don't use the fixed userID anymore.

File: src/containers/Profile/index.js [hunk 2]
```diff
@@ -3,23 +3,25 @@ import React from 'react';
...
       {component.componentBody}
     </ProfileComponent>
   ));
   return (
     <PageTemplate heading="Profile">
-      <div className="Profile">
-        {renderComponents}
-        <AssignmentMain />
-      </div>
+      <div className="Profile">{renderComponents}</div>
```
Comment (trungh13) [inline src/containers/Profile/index.js]: Yasir commited his part with AssignmentMain already, please resolve it first !

Comment (bojanmitic) [reply src/containers/Profile/index.js]: It's resolved, i render this component in the object above.

Comment (trungh13) [reply src/containers/Profile/index.js]: my bad

**Pull request merged**